Bundles/Raven.Client.Authorization/Raven.Client.Authorization.xproj

```diff
@@ -9,7 +9,7 @@
     <ProjectGuid>43ebd466-4a7b-4cf3-85cd-72fecee22986</ProjectGuid>
     <RootNamespace>Raven.Client.Authorization</RootNamespace>
     <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)'=='' ">..\..\artifacts\obj\$(MSBuildProjectName)</BaseIntermediateOutputPath>
-    <OutputPath Condition="'$(OutputPath)'=='' ">..\..\artifacts\bin\$(MSBuildProjectName)\</OutputPath>
+    <OutputPath Condition="'$(OutputPath)'=='' ">.\bin\</OutputPath>
```

Comment:
This likely just changed as I was working around files in Visual Studio. Nothing intended.